### PR TITLE
Add reducer action

### PR DESCRIPTION
**Pull request by SimonGideon**
- Define action types for adding and removing a book.
- Set the initial state to be an empty array of books.
- Export Action Creators for your actions.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const CHECK_STATUS = 'Bookstore/categories/UNDER_CONSTRUCTION';
```
Comment (emmiiorji) [inline src/redux/categories/categories.js:1]: In order to completely follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) as stated in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/5801583dd2ad132a9326d4bd429291d954e4e824/bookstore/add_reducers_an_actions.md?plain=1#L42), kindly modify the action types to have the form `npm-module-or-app/reducer/ACTION_TYPE`.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'Bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'Bookstore/books/REMOVE_BOOK';
```
Comment (emmiiorji) [inline src/redux/books/books.js:2]: Just as pointed out above, in order to completely follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) as stated in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/5801583dd2ad132a9326d4bd429291d954e4e824/bookstore/add_reducers_an_actions.md?plain=1#L42), kindly modify the action types to have the form npm-module-or-app/reducer/ACTION_TYPE.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @SimonGideon, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to use of GitHub flow
- Reducers and Actions are properly set up


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 1 above)
Comment (SimonGideon) [reply src/redux/categories/categories.js]: @emmiiorji Thanks for the review. Tha requested changes are corrected

**Review by Gambit142 — APPROVED** [review]
Hi @SimonGideon 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Good use of _**duck**_ pattern

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**